app/helpers/host_aspects_helper.rb

```diff
@@ -0,0 +1,29 @@
+module HostAspectsHelper
+  # override host's tabs to add all aspects as tabs
+  def host_additional_tabs(host)
+    base_tabs = {}
+    aspect_tabs = load_aspects(host)
+    base_tabs.merge!(aspect_tabs)
```

Comment:
if you are trying to minimize local state (like here: https://github.com/theforeman/foreman/pull/2665/files#diff-13b828e2868627bafc7af5e036628234R7), why not: `base_tabs.merge!(load_aspects(host))`